validator.py

```diff
@@ -872,6 +886,72 @@ def verify_allowed_actions(self):
                         self.logger.log(LogLevel.WARN, f"{x['action_type']} is a restricted action at scene with id '{scenes[i]['id']}', but appears in the action_mapping within that scene.")
                         self.invalid_values += 1
 
+    def pulse_ox_info(self, scene_id):
+        self.invalid_values += 1
```

Comment:
We'll need to change this to warning_count once my branch is merged into yours

Reply:
Oh that's a good point. I didn't think about that. @Anchita14 do what Darren said here. Let me know if you need any help with it